Sources/RswiftCore/Generators/ColorStructGenerator.swift

```diff
@@ -57,8 +57,13 @@ struct ColorStructGenerator: ExternalOnlyStructGenerator {
       implements: [],
       typealiasses: [],
       properties: colorLets,
-      functions: groupedColors.uniques.map { [ generateColorFunction(for: $0, at: externalAccessLevel, prefix: qualifiedName),
-                                               generateWatchOSColorFunction(for: $0, at: externalAccessLevel, prefix: qualifiedName)] }.flatMap { $0 },
+      functions: groupedColors.uniques.map { name -> [Function] in
+        var functions = [ generateColorFunction(for: name, at: externalAccessLevel, prefix: qualifiedName)]
+        if case .hostingBundle = bundle {
+          functions.append(generateWatchOSColorFunction(for: name, at: externalAccessLevel, prefix: qualifiedName))
+        }
+        return functions
+      }.flatMap { $0 },
```

Comment:
Curious if we can flatMap directly here like this:
```suggestion
      functions: groupedColors.uniques.flatMap { name -> [Function] in
        var functions = [ generateColorFunction(for: name, at: externalAccessLevel, prefix: qualifiedName)]
        if case .hostingBundle = bundle {
          functions.append(generateWatchOSColorFunction(for: name, at: externalAccessLevel, prefix: qualifiedName))
        }
        return functions
      },
```